OneSignalSDK/onesignal/src/main/java/com/onesignal/LocationGMS.java

```diff
@@ -85,51 +86,82 @@
       void complete(LocationPoint point);
    }
 
+   abstract static class LocationAndPromptHandler implements LocationHandler {
+      void onPromptHandle(boolean accepted) {}
```

Comment:
Rename to `onAnswered`  to make more sense when calling `locationPromptHandler.onAnswered(boolean)`